### PR TITLE
Feature/devise

### PR DESCRIPTION
**Pull request by fherrerao**
# Hi There 💯 
### Features in this branch:

- Register a new user.
- User logs in with a combination of email and password.
- Hashed password was stored in the database.
- Ask for confirmation of email.
- Reset password functionality.

Please let us know if we should make changes.

### CODE REVIEW

File: app/views/devise/sessions/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
+</div>
```
Comment (TedLivist) [inline app/views/devise/sessions/new.html.erb]: - [ ] Kindly ensure your login page design matches the wireframe.

  Your Implementation | Provided Design
  :---:|:---:
  xx | otoz-1

File: app/views/devise/registrations/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      <%= f.submit "Sign up" %>
+    </div>
+
+  </div>
+
+  <% end %>
+
+  
+
+<%= render "devise/shared/links" %>
```
Comment (TedLivist) [inline app/views/devise/registrations/new.html.erb]: - [ ] Kindly ensure the signup page follows the design as well

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @fherrerao,

Good job so far!
✔️ Good documentation
✔️ Installation of Devise
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi 👋 @fherrerao, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### You have done a great job working on the requested changes ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**